util/dvsim/BUILD

```diff
@@ -0,0 +1,258 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("@rules_python//python:defs.bzl", "py_binary")
+load("@ot_python_deps//:requirements.bzl", "requirement")
+
+package(default_visibility = ["//visibility:public"])
+
+py_library(
+    name = "utils",
+    srcs = [
```

Comment:
Nit: I tend to put single-file source lists on one line; helps avoid noise.

Reply:
fixed.